cfme/infrastructure/pxe/ui.py

```diff
@@ -0,0 +1,36 @@
+from base import PXEBase
+from utils.appliance import ViaUI, current_appliance
+
+from cfme.exceptions import CandidateNotFound
+from selenium.common.exceptions import NoSuchElementException
+from functools import partial
+import cfme.web_ui.toolbar as tb
+import cfme.web_ui.accordion as acc
+
+cfg_btn = partial(tb.select, 'Configuration')
+
+pxe_tree = partial(acc.tree, "PXE Servers", "All PXE Servers")
+
+current_appliance.menu.add_branch('infrastructure_pxe',
```

Comment:
This "wording" gives me a feeling that once you push a new appliance on the stack, it will have no menu ...